.github/ISSUE_TEMPLATE/GC-Articles-Publishing

```diff
@@ -0,0 +1,57 @@
+---
+name: GC Articles Publishing User Story Template
+about: Template for creating new User Stories for Notify backlog
+title: 'Give your user story a simple, clear title'
+labels: ''
+assignees: ''
+
+---
+## Description
+As a (user), I need to be able to do (X) so that I can achieve (Y) outcome. 
+
+WHY are we building?
+WHAT are we building?
+VALUE created by our solution
+
+## Documentation and Artifacts
+Good docs, figma mockups, ADRs, screenshots etc.
+
+## Acceptance Criteria
+Given some context, when (X) action occurs, then (Y) outcome is achieved 
+
+- [ ] Cypress UI tests if needed.
+- [ ] Generate appropriate log messages so that executions of this feature can be tracked
+- [ ] Can misuse of this feature cause harm? If yes, create an alert
+- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
+- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)
+
+* A11y
+* Bilingualism
+* Privacy considerations
+* Security controls in place
+* Measuring success and metrics
+
+## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
+- (find stuff yourself)
+
+## QA Steps
+- [ ] Tested in a realistic production scenario
+
+## GC Articles Publish checklist
+- [ ]  Supporting feature is released and unflagged
+- [ ]  Create or update slug
+- [ ]  Page content and slug is translated
+- [ ]  Document links to and from the page
+- [ ]  Add page slug and title to [inventory](https://docs.google.com/document/d/1N6B4fSwoM40m-pDLI70BvLfL7O0n04ndyAXYaIwNbNw/edit)
+- [ ]  If new page, add WAF rule
+- [ ]  Create new page or update existing page content
+- [ ]  Create or update translation to the page
+- [ ]  Link translations by choosing an assigned translation
+- [ ]  If new page, save as draft
+- [ ]  If existing page, update the page
+- [ ]  Post page in Slack for last minute reviews
+- [ ]  On staging: Clear the cache through GC Notify's admin panel
+- [ ]  On staging: Check links, images and language toggle
+- [ ]  Give a heads up to the app-team channel once ready to publish
+- [ ]  On staging and production: Clear the cache through GC Notify's admin panel
+- [ ]  Clear Cloudfront cache in GCA as needed (Andrew and Phil are GCA admins)
```

Comment:
Only Pat has permissions to do this. And I think we haven't needed to do this for quite a while, presumably after some maintenance updates to GC Articles. We can remove!